lightningd/peer_htlcs.c

```diff
@@ -873,7 +873,7 @@ htlc_accepted_hook_callback(struct htlc_accepted_hook_payload *request,
 	tal_free(request);
 }
 
-REGISTER_PLUGIN_HOOK(htlc_accepted, PLUGIN_HOOK_SINGLE,
+REGISTER_PLUGIN_HOOK(htlc_accepted, PLUGIN_HOOK_CHAIN,
```

Comment:
From commit msg:

> Changelog-Changed: plugin: The `htlc_accepted` hook is not a chained hook. Multiple plugins can now register to handle incoming HTLCs and they will be called in order until one returns a result that isn't `{"result": "continue"}`, at which point the HTLC is considered handled.

s/not/now/.

Changelog is not a place for documentation!  How about simply:

Changelog-Changed: plugin: Multiple plugins can now register for the `htlc_accepted` hook.